src/plugins/data/server/search/aggs/aggs_service.ts

```diff
@@ -72,8 +74,18 @@ export class AggsService {
         const getConfig = <T = any>(key: string): T => {
           return uiSettingsCache[key];
         };
+        const isDefaultTimezone = () => !uiSettingsClient.isOverridden('dateFormat:tz');
```

Comment:
On line 102 `isDefaultTimezone` is already defined. I can't remember the details off the top of my head, but at the time I determined `isOverridden` was not the behavior we were looking for and we needed to specifically check for whether it was `Browser`.

Was there a particular reason you had used `isOverridden` here? If not I'd recommend we stick with the existing implementation and use it both places. (I can also dig into this and see if I can remember exactly what was going on).  😄

Reply:
Ah, missed that, thanks. Better not risk anything 🙂